ui/nuxeo-data-table/data-table-checkbox.js

```diff
@@ -68,7 +68,11 @@ import '../widgets/nuxeo-checkmark.js';
 
     ready() {
       super.ready();
-      this.setAttribute('role', 'cell');
+      if (this.header !== (this.disabled || !this.disabled)) {
```

Comment:
What are you trying to compare here? `disabled` is a boolean, if you do `this.disabled || !this.disabled`, this will always true 🤔 
`true || !true = true || false = true` or `false || !false = false || true = true`